docker-app/qfieldcloud/core/models.py

```diff
@@ -1083,6 +1083,17 @@ class PackagingOffliner(models.TextChoices):
         QGISCORE = "qgiscore", _("QGIS Core Offline Editing (deprecated)")
         PYTHONMINI = "pythonmini", _("Optimized Packager")
 
+    def get_localized_layers(self) -> list[dict]:
+        """
+        Returns a list of all localized layers (as dictionaries)
+        from project_details['layers_by_id'].
+        """
+        layers = (
+            self.project_details.get("layers_by_id", {}) if self.project_details else {}
+        )
```

Comment:
```suggestion
        if self.project_details:
            layers_by_id = self.project_details.get("layers_by_id", {}) 
        else:
            layers_by_id = {}
```
Avoid renaming keys. Also avoid inline ifs.